libs/designsystem/package.json

```
@@ -6,7 +6,8 @@
     "@ionic/angular": "5.5.2",
     "highcharts": "8.0.4",
     "moment": "2.24.0",
-    "@fontsource/roboto": "4.2.1"
+    "@fontsource/roboto": "4.2.1",
+    "inputmask": "^5.0.5"
```

Consider using exact version as we do for other deps:
```suggestion
    "inputmask": "5.0.5"
```